code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,649 @@
+	cultists_required = 2
+	icon_state = "5"
+	color = rgb(0, 255, 0)
+
+/obj/effect/rune/summon/invoke(mob/living/user)
+	var/list/cultists = list()
+	for(var/datum/mind/M in ticker.mode.cult)
+		cultists.Add(M.current)
+	var/mob/living/cultist_to_summon = input("Who do you wish to call to [src]?", "Followers of the Geometer") as null|anything in (cultists - user)
+	if(!cultist_to_summon)
+		user << "<span class='warning'>You require a summoning target!</span>"
+		fail_invoke()
+		log_game("Summon Cultist rune failed - no target")
 		return
-
-
-///Armor: The user will be armed with cultist garb, a trophy rack, and a high-damage sword. If items are in these slots, they will not be equipped and simply deleted.
-
-
-/obj/effect/rune/proc/armor()
-	var/list/armors = list("zealot")
-	var/armorworn
-	var/mob/living/carbon/human/user = usr
-	if(istype(user.wear_suit, /obj/item/clothing/suit/wizrobe) && istype(user.head, /obj/item/clothing/head/wizard))
-		armors += "summoner"
-	if(istype(user.wear_suit, /obj/item/clothing/suit/space) && istype(user.head, /obj/item/clothing/head/helmet/space))
-		armors += "traveler"
-	if(istype(user.wear_suit, /obj/item/clothing/suit/space/captain) && istype(user.head, /obj/item/clothing/head/helmet/space/captain))
-		armors += "marauder"
-	if(istype(user.wear_suit, /obj/item/clothing/suit/armor/reactive))
-		armors += "trickster"
-	if(istype(user.wear_suit, /obj/item/clothing/suit/toggle/labcoat/cmo))
-		armors += "physician"
-	if(istype(src,/obj/effect/rune))
-		armorworn = input ("Choose your attire.") in armors
-	else
-		armorworn = "zealot"
-	if (armorworn == "zealot")
-		usr.say("N'ath reth sh'yro eth d[pick("'","`")]raggathnor!")
-		user.equip_to_slot_or_del(new /obj/item/clothing/head/culthood/alt(user), slot_head)
-		user.equip_to_slot_or_del(new /obj/item/clothing/suit/cultrobes/alt(user), slot_wear_suit)
-		user.equip_to_slot_or_del(new /obj/item/clothing/shoes/cult(user), slot_shoes)
-		user.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
-		user.put_in_hands(new /obj/item/weapon/melee/cultblade(user))
-		usr.visible_message("\red The rune disappears with a flash of red light, and a set of armor appears on [usr]...", \
-		"\red You are blinded by the flash of red light! After you're able to see again, you see that you are now wearing a set of armor.")
-	if (armorworn == "summoner" || armorworn == "trickster" || armorworn == "physician")
-		usr.say("Uln Shogg Hafh[pick("'","`")]drn!")
-		user << "\red You quietly prick your finger and make a pact with the geometer of blood to acquire more power."
-		user.take_overall_damage(rand(5,20))
-		qdel(user.head)
-		qdel(user.wear_suit)
-		user.equip_to_slot_or_del(new /obj/item/clothing/head/magus(user), slot_head)
-		user.equip_to_slot_or_del(new /obj/item/clothing/suit/magusred(user), slot_wear_suit)
-		user.equip_to_slot_or_del(new /obj/item/clothing/shoes/cult(user), slot_shoes)
-		user.spellremove(user)
-		usr.visible_mes
```

Comment:
Haha, yeah uh, path2text() doesn't exist because it's simple and I was tired when I wrote that bit.
path2text() is basically just ```"[thing.type]"```

"Rune2Talisman"
```
var/Rtype = "[R.type]"
var/list/Rsplit = text2list(Rtype,"/")
var/RuneType = Rsplit[4]
var/TailismanType = "/obj/item/weapon/paper/talisman/[RuneType]"
if(ispath(TalisimanType))
    new TalismanType(get_turf(src))
else
    new /obj/item/weapon/paper/talisman/malformed (get_turf(src))
```